Assets/Controllers/GameManager.cs

```diff
@@ -208,7 +226,24 @@ public void GetFirstPawnForActivePlayerPositionAndFocusCamera()
             Camera.main.GetComponent<CameraMotionHandler>().MoveToPosition(pawnPosition);
         } else
         {
-            //probably make the End Turn button flash yellow or something
+            SuggestEndingTurn();
         }
     }
+
+    public void SetActivePlayer(Player player)
+    {
+        activePlayer = player;
+        player.StartTurn();
+        worldDisplay.DisplayMapForPlayer(player);
+    }
+
+    public WorldDisplay GetWorldDisplay()
+    {
+        return worldDisplay;
+    }
+
+    void SuggestEndingTurn()
+    {
+        print("You have no pawns left with turns!");
```

Comment:
Should this be a user notification? Or maybe it is? Does this print to a console?